frontend/components/product-list/sections/FilterableProductsSection/ProductListFilters/FilterRow.tsx

```diff
@@ -266,13 +245,74 @@ const Row = React.memo(
    })
 );
 
-function parseRange(value: string): [number, number] {
-   const [min, max] = value.split(':');
-   return [parseFloat(min), parseFloat(max)];
+interface PriceFilterProps {
+   priceRangeFacet: Facet;
+   showAllFacetValues: boolean;
+   onResize(): void;
+}
+
+function PriceFilter({
+   priceRangeFacet,
+   showAllFacetValues,
+   onResize,
+}: PriceFilterProps) {
+   const priceFacet = useFacet('price');
+   return (
+      <>
+         <ListFilter
+            key={priceRangeFacet.handle}
+            facet={priceRangeFacet}
+            multiple
+            dependentFacets={[priceFacet.handle]}
+            showAllValues={showAllFacetValues}
+            sortItems={sortByPriceRange}
+         />
+         <RangeFilterInput
+            facet={priceFacet}
+            minFieldPrefix="$"
+            minFieldPlaceholder="Min"
+            maxFieldPrefix="$"
+            maxFieldPlaceholder="Max"
+            dependentFacets={[priceRangeFacet.handle]}
+            onError={onResize}
+            onDismissError={onResize}
+         />
+      </>
+   );
+}
+
+const priceRangeRegex = /[^\d]*(\d+)\s*-\s*[^\d]*(\d+)/i;
```

Comment:
I might make these a bit more tolerant of changes in format: `/(\d+)^\D+(\d+)/`. With regexes being greedy, this ought to work fine: https://regexr.com/6h2j7

Also, `\D` is more succinct than `[^\d]` and means the same thing:
![image](https://user-images.githubusercontent.com/26855/157555250-b71c413b-fb6c-448b-a241-1aca1ab8ba03.png)